### PR TITLE
chore(css): alpha order and clean up css

### PR DESCRIPTION
**Pull request by chriswhawkins**
Necessary? Not really. But I feel a little better.

Alpha order properties. 
Remove trailing whitespace. 
Remove 0's before decimals.
Trailing semicolons for all properties.

Closes: #58

### CODE REVIEW

File: assets/css/default.css [hunk 1]
```diff
@@ -71,39 +71,39 @@ a:hover {
...
   background-position: center;
```
Comment (tborres) [inline assets/css/default.css]: Webstorm recommends "optimizing" background properties here as `background: url(data:image/gif;base64,R0lGODlhAQABAIAAANbW1gAAACH/C1hNUCBEYXRhWE1QPD94cGFja2V0IGJlZ2luPSLvu78iIGlkPSJXNU0wTXBDZWhpSHpyZVN6TlRjemtjOWQiPz4gPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iQWRvYmUgWE1QIENvcmUgNS4wLWMwNjEgNjQuMTQwOTQ5LCAyMDEwLzEyLzA3LTEwOjU3OjAxICAgICAgICAiPiA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPiA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtbG5zOnhtcE1NPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvbW0vIiB4bWxuczpzdFJlZj0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL3NUeXBlL1Jlc291cmNlUmVmIyIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1LjEgTWFjaW50b3NoIiB4bXBNTTpJbnN0YW5jZUlEPSJ4bXAuaWlkOjhDRDk0RUI1NkQ3NzExRTFCRTBBQUNBMjU2RUU0Q0U1IiB4bXBNTTpEb2N1bWVudElEPSJ4bXAuZGlkOjhDRDk0RUI2NkQ3NzExRTFCRTBBQUNBMjU2RUU0Q0U1Ij4gPHhtcE1NOkRlcml2ZWRGcm9tIHN0UmVmOmluc3RhbmNlSUQ9InhtcC5paWQ6OENEOTRFQjM2RDc3MTFFMUJFMEFBQ0EyNTZFRTRDRTUiIHN0UmVmOmRvY3VtZW50SUQ9InhtcC5kaWQ6OENEOTRFQjQ2RDc3MTFFMUJFMEFBQ0EyNTZFRTRDRTUiLz4gPC9yZGY6RGVzY3JpcHRpb24+IDwvcmRmOlJERj4gPC94OnhtcG1ldGE+IDw/eHBhY2tldCBlbmQ9InIiPz4B//79/Pv6+fj39vX08/Lx8O/u7ezr6uno5+bl5OPi4eDf3t3c29rZ2NfW1dTT0tHQz87NzMvKycjHxsXEw8LBwL++vby7urm4t7a1tLOysbCvrq2sq6qpqKempaSjoqGgn56dnJuamZiXlpWUk5KRkI+OjYyLiomIh4aFhIOCgYB/fn18e3p5eHd2dXRzcnFwb25tbGtqaWhnZmVkY2JhYF9eXVxbWllYV1ZVVFNSUVBPTk1MS0pJSEdGRURDQkFAPz49PDs6OTg3NjU0MzIxMC8uLSwrKikoJyYlJCMiISAfHh0cGxoZGBcWFRQTEhEQDw4NDAsKCQgHBgUEAwIBAAAh+QQAAAAAACwAAAAAAQABAAACAkQBADs=) repeat-x center;` 

Though that may hurt readability.

File: assets/css/default.css [hunk 2]
```diff
@@ -111,10 +111,10 @@ a:hover {
 
 #banner {
   background-color: #4F5052;
-  background-image: url(/assets/img/banner.jpg);
+  background-image: url('/assets/img/banner.jpg');
```
Comment (tborres) [inline assets/css/default.css]: Optimize background props? 
`background: #4F5052 url('/assets/img/banner.jpg') no-repeat center;`

**Comment by tborres** [comment]
:+1: Looks good. 2 webstorm suggestions

**Comment by chriswhawkins** [comment]
@tborres Thanks, I updated to bg shortcode and lowercased the colors. Merging...

**Pull request merged**